### PR TITLE
v1.3.10 - Added getProfile and updateProfile commands to provision proxy

### PR DESCRIPTION
**Pull request by tsmith1985**

### CODE REVIEW

**Review by rmomii — APPROVED** [review]
other than the history.md and package.json merge conflicts, it looks good

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "playnetwork-sdk",
-  "version": "1.3.8",
+  "version": "1.3.10",
```
Comment (ggentzel) [inline package.json]: why did we skip over 1.3.9 here?

**Review by ggentzel — APPROVED** [review]
Just looks like it needs to pull from develop to get sync'd and a question about why the package version skipped over a patch increment

File: lib/provision.js [hunk 2]
```diff
@@ -137,57 +228,78 @@ module.exports = function (provisionOptions, ensureAuthHeaders, self) {
 		return validation.promiseOrCallback(exec, callback);
 	};
 
-	self.checkUpdate = (options, callback) => {
+	self.createProfile = (profile, callback) => {
 		// handle any non-specified input params
-		if (typeof options === 'function') {
-			callback = options;
-			options = undefined;
+		if (typeof order === 'function') {
```
Comment (ggentzel) [inline lib/provision.js]: should order be profile instead, copypasta?

File: package.json (hunk 1 above)
Comment (ggentzel) [reply package.json]: disregard this comment and the thing about needing pulling. i see what happened there now. i think just the one comment below that order should probably be profile copypaste issue need addressing

File: lib/provision.js (hunk 2 above)
Comment (tsmith1985) [reply lib/provision.js]: 😜

**Pull request merged**